src/app/clusters/scenes/scenes.cpp

```diff
@@ -469,29 +429,6 @@ bool emberAfScenesClusterGetSceneMembershipCallback(chip::app::Command * command
         }
     }
 
-    if (commandObj == nullptr)
-    {
-        // The status, capacity, and group id are always included in the response, but
-        // the scene count and scene list are only included if the group id matched.
-        emberAfFillExternalBuffer(
-            (ZCL_CLUSTER_SPECIFIC_COMMAND | ZCL_FRAME_CONTROL_SERVER_TO_CLIENT | EMBER_AF_DEFAULT_RESPONSE_POLICY_RESPONSES),
-            ZCL_SCENES_CLUSTER_ID, ZCL_GET_SCENE_MEMBERSHIP_RESPONSE_COMMAND_ID, "uuv", status,
-            (EMBER_AF_PLUGIN_SCENES_TABLE_SIZE - emberAfPluginScenesServerNumSceneEntriesInUse()), // capacity
-            groupId);
-
-        // Get Scene Membership commands are only responded to when they are
-        // addressed to a single device or when an entry in the table matches.
-        if (emberAfCurrentCommand()->type == EMBER_INCOMING_UNICAST ||
-            emberAfCurrentCommand()->type == EMBER_INCOMING_UNICAST_REPLY || sceneCount != 0)
-        {
-            sendStatus = emberAfSendResponse();
```

Comment:
If we're removing this, why do we still need the `emberAfPutInt8uInResp` bits above?